src/test/java/SeperateIntegerTest.java

```diff
@@ -0,0 +1,16 @@
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SeperateIntegerTest {
```

Comment:
```suggestion
class SeperateIntegerTest {
```
junit 5부터는 테스트 클래스에 pulibc을 제거해주셔도 됩니다.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-classes-and-methods